.github/workflows/builds.yml

```diff
@@ -2460,7 +2462,7 @@ jobs:
       - run: tar xf prep/suricata-update.tar.gz
       - run: tar xf prep/suricata-verify.tar.gz
       - run: ./autogen.sh
-      - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure --enable-warnings --enable-unittests
+      - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure --enable-warnings --enable-unittests --enable-ebpf --enable-ebpf-build
```

Comment:
Add these also to the `DISTCHECK_CONFIGURE_FLAGS`.  I don't think we had that before, but it would be a good thing to add here.